src/MLOps.NET.SQLite/MLOpsBuilderExtensions.cs

```diff
@@ -13,7 +13,7 @@ public static class MLOpsBuilderExtensions
         /// <param name="builder">MLOpsBuilder to add Azure Storage providers to</param>
         /// <param name="destinationFolder">Destination folder (optional with default value of C:\MLops)</param>
         /// <returns>Provided MLOpsBuilder for chaining</returns>
-        public static MLOpsBuilder UseSQLite(this MLOpsBuilder builder, string destinationFolder)
+        public static MLOpsBuilder UseSQLite(this MLOpsBuilder builder, string destinationFolder = @"C:\MLops")
```

Comment:
On Linux systems, it creates a weird folder relative to bin folder `C/\MLops`

Reply:
@shanranm that's a great point. Would it make sense not to have a default for that reason? If you override this value and run it on Linux, does that work?